services/wallet/datastore.go

```diff
@@ -561,7 +561,7 @@ var (
 )
 
 // LinkWallet links a wallet together
-func (pg *Postgres) LinkWallet(ctx context.Context, ID string, userDepositDestination string, providerLinkingID uuid.UUID, anonymousAddress *uuid.UUID, depositProvider, country string) error {
+func (pg *Postgres) LinkWallet(ctx context.Context, ID string, userDepositDestination string, providerLinkingID uuid.UUID, depositProvider string) error {
```

Comment:
Similar nitpick as above, but would require more unrelated changes – the code uses `ID` and then later creates `id` from it. 🤦‍♂️

Reply:
added to the wallet refactor ticket